ui/src/app/cohort-search/cohort-search/cohort-search.component.tsx

```diff
@@ -437,25 +425,7 @@ export const CohortSearch = withCurrentCohortSearchContext()(class extends React
             : <React.Fragment>
               {loadingSubtree && <SpinnerOverlay/>}
               <div style={loadingSubtree ? {height: '100%', pointerEvents: 'none', opacity: 0.3} : {height: '100%'}}>
```

Comment:
Sorry, missed this the first time around. These lines will also need to move over to `CriteriaSearch` to show a spinner and disable the tree while loading.